content/layering/layering.adoc

```diff
@@ -8,7 +8,31 @@
 :numbered:
 :homepage https://github.com/projectatomic/container-best-practices:
 
-Guidelines on creating layers.
+This chapter provides guidelines on creating layers.
+
+== Minimizing the Number of Layers
+
+To keep your Dockerfiles easily readable and maintainable in future, you should maintain a reasonable number of layers. In general, a new layer is created for every new instruction defined. However, this does not necessarily mean that one instruction should be associated with only one command or definition.
```

Comment:
Layering is not about making Dockerfiles more readable, actually it makes them less readable as you are collapsing groups of commands together to keep the number of layers created small.